app/assets/stylesheets/components/internal_link.scss

```diff
@@ -12,4 +12,22 @@
     background-repeat: no-repeat;
     background-position: 0 .25em;
   }
+}
+
+// Keep from being hidden behind nav bar on small screens.
+@media (max-width: 992px) {
+  .course-section__title::before, .lesson-content h3::before {
```

Comment:
We use the [BEM](http://getbem.com/) naming convention to organize of styles into components. A part of that is keeping all styles specific to that component within its component file so its easier to find and extend.

Targeting `.course_section__title` and the `lesson_content` within the internal-link file is violating that.

We could instead create a internal-link-offset mixin in the mixins stylesheet:
```
@mixin internal-link-offset() {
  display: block;
  content: " ";
  margin-top: -100px;
  height: 100px;
  visibility: hidden;
  pointer-events: none;
}
```
This will allow us to then share these styles in the lesson_content and course sections component:

lesson_content:
```
h3 {
    margin-bottom: 20px;
    margin-top: 50px;
    font-weight: bold;
    color: $text-primary;
    font-size: 1.5em;

    @include media-breakpoint-down(md) {
      &:before {
        @include internal-link-offset();
      }
    }
  }
```

course_sections:
```
&__title {
    font-size: 1.5em;
    letter-spacing: 0.3px;
    margin-bottom: 16px;
    font-weight: 600;

    @include media-breakpoint-down(md) {
      &:before {
        @include internal-link-offset();
      }
    }
  }
```